xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountTransactionsRequestParams.java

```diff
@@ -38,10 +38,69 @@
    *
    * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
    */
+  @Deprecated
   static ImmutableAccountTransactionsRequestParams.Builder builder() {
     return ImmutableAccountTransactionsRequestParams.builder();
   }
 
+  /**
+   * Construct a builder for this class with {@link LedgerIndex} field set.
+   *
+   * @param ledgerIndex An index of type {@link LedgerIndex} to specify a particular ledger.
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(LedgerIndex ledgerIndex) {
+    return ImmutableAccountTransactionsRequestParams.builder().ledgerSpecifier(
+        Optional.of(LedgerSpecifier.of(ledgerIndex))
+    );
+  }
+
+  /**
+   * Construct a builder for this class with {@link LedgerSpecifier} field set.
+   *
+   * @param ledgerSpecifier A specifier of type {@link LedgerSpecifier} to specify a ledger.
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
+   */
+  static ImmutableAccountTransactionsRequestParams.Builder builder(LedgerSpecifier ledgerSpecifier) {
+    return ImmutableAccountTransactionsRequestParams.builder().ledgerSpecifier(Optional.ofNullable(ledgerSpecifier));
+  }
+
+  /**
+   * Construct a builder for this class with ledgerHash field set.
+   *
+   * @param ledgerHash Hash of type {@link Hash256} of a specific ledger.
+   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
```

Comment:
There should be an empty javadoc line between `@param`s and `@return`. This isn't a hard and fast rule, but we should try to be consistent across the library to reduce PR noise.

You should be able to use IntelliJ's "reformat file" tool (`option + command + L` on mac).

```suggestion
   * @param ledgerHash Hash of type {@link Hash256} of a specific ledger.
   *
   * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
```